Ports/zsh/zshrc

```diff
@@ -0,0 +1,63 @@
+###############
+# Completions #
+###############
+
+# Completions
+autoload -U compinit
+compinit -C
+
+# Arrow key menu for completions
+zstyle ':completion:*' menu select
+
+# Case-insensitive (all),partial-word and then substring completion
+zstyle ':completion:*' matcher-list 'm:{a-zA-Z}={A-Za-z}' 'r:|[._-]=* r:|=*' 'l:|=* r:|=*'
+
+###########
+# Aliases #
+###########
+
+# Set up aliases
+alias mv='nocorrect mv'       # no spelling correction on mv
+alias cp='nocorrect cp'       # no spelling correction on cp
+alias mkdir='nocorrect mkdir' # no spelling correction on mkdir
+alias j=jobs
+alias pu=pushd
+alias po=popd
+alias d='dirs -v'
+alias h=history
+alias grep=egrep
+alias ll='ls -l'
+alias la='ls -a'
```

Comment:
In my opinion, if we want to add some default aliases, we should ideally mirror at most the ones that already exist in our own Shell's rc file.

In fact, that should probably apply to all settings that we could set here that have equivalent functionality in the default system.

Reply:
Ok, I left in a single alias, basic autocompletion and history management as well as the fix for the prompt.